source/01-global/normalize/_normalize.scss

```diff
@@ -53,11 +53,22 @@ select {
   text-transform: none;
 }
 
+// Styled to look like a link.
 button,
 [type='button'],
 [type='reset'],
 [type='submit'] {
-  -webkit-appearance: button;
+  -webkit-appearance: none;
+  border: 0;
+  border-radius: 0;
+  box-shadow: none;
+  cursor: pointer;
+  font-weight: gesso-font-weight(regular);
+  margin: 0;
+  overflow: visible;
+  padding: 0;
+  text-align: left;
+  text-decoration: underline;
```

Comment:
I added a text-button mixin that you can use here to make buttons look like regular text links: https://github.com/forumone/gesso_next/blob/main/source/00-config/mixins/_button.scss#L101-L119

Reply:
Thanks! I switched to using that mixin to account for a bunch of the default button styles.